src/content/docs/infrastructure/elastic-container-service-integration/understand-use-ecs-data.mdx

```diff
@@ -48,6 +48,10 @@ SELECT uniqueCount(containerId)
   FACET imageName SINCE 1 HOUR AGO
 ```
 
+<Callout variant="important">
+On EC2 nodes with Cgroup V2 version the `nri-docker` integration collects all container metrics from the Docker API. Variations in collected metrics are elaborated in the [`ContainerSample`](/attribute-dictionary/?event=ContainerSample) attributes.
+</Callout>
+
```

Comment:
Shall we explain a bit why this is the way metrics are fetched on EC2 nodes with cgroups v2? The callout can be a bit surprising since it is the only explanation concerning how container metrics are fetched. We could start the callout with something like:

```
While on EC2 nodes with Cgroups V1 Container metrics are fetched using the Docker API in conjuction with /proc filesystem data, on nodes with Cgroups V2 metrics are collected using the Docker API only because (...)
```

WDYT?

Reply:
I tried to explained in a simplified way but i couldn't do it without generating more doubts and adding confusion TBH.